libs/design/src/atoms/form/form-field/form-field/form-field.component.html

```diff
@@ -1,7 +1,24 @@
-<div class="daff-form-field__control" [class.daff-error]="isError" [class.daff-valid]="isValid" [class.daff-focus]="isFocused">
-  <ng-content></ng-content>
-  <div class="daff-form-field__icon" *ngIf="_control.controlType === 'native-select'">
-    <fa-icon [icon]="faChevronDown"></fa-icon>
-  </div>
+<div
+	class="daff-form-field__control"
+	[class.daff-focus]="isFocused"
+	[class.daff-error]="isError"
+	[class.daff-valid]="isValid"
+	[class.daff-filled]="isFilled"
+>
+  <span class="daff-form-field__icon daff-form-field__icon--prefix">
+    <ng-content select=".prefix"></ng-content>
+  </span>
+	<ng-content></ng-content>
+	<div
+		class="daff-form-field__icon"
+		*ngIf="_control.controlType === 'native-select'"
+	>
+		<fa-icon [icon]="faChevronDown"></fa-icon>
+	</div>
+  <span class="daff-form-field__icon daff-form-field__icon--suffix">
+    <ng-content select=".suffix"></ng-content>
+  </span>
 </div>
-<ng-content select="daff-error-message"></ng-content>
+@if (_control?.ngControl?.touched) {
+  <ng-content select="daff-error-message"></ng-content>
+}
```

Comment:
`hint` should be added as a `<ng-content>` outside of `daff-form-field__control`